packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,276 @@
+  { label: "Taxi Driver", year: 1976 },
+  { label: "Lawrence of Arabia", year: 1962 },
+  { label: "Double Indemnity", year: 1944 },
+  {
+    label: "Eternal Sunshine of the Spotless Mind",
+    year: 2004,
+  },
+  { label: "Amadeus", year: 1984 },
+  { label: "To Kill a Mockingbird", year: 1962 },
+  { label: "Toy Story 3", year: 2010 },
+  { label: "Logan", year: 2017 },
+  { label: "Full Metal Jacket", year: 1987 },
+  { label: "Dangal", year: 2016 },
+  { label: "The Sting", year: 1973 },
+  { label: "2001: A Space Odyssey", year: 1968 },
+  { label: "Singin' in the Rain", year: 1952 },
+  { label: "Toy Story", year: 1995 },
+  { label: "Bicycle Thieves", year: 1948 },
+  { label: "The Kid", year: 1921 },
+  { label: "Inglourious Basterds", year: 2009 },
+  { label: "Snatch", year: 2000 },
+  { label: "3 Idiots", year: 2009 },
+  { label: "Monty Python and the Holy Grail", year: 1975 },
+];
+
+const Template: ComponentStory<typeof Autocomplete> = (args) => {
+  return (
+    <Autocomplete
+      {...args}
+      options={top100Films}
+      renderInput={(params) => (
+        <TextField
+          {...params}
+          endAdornment={params.InputProps.endAdornment}
+          hint="Select your favorite movie"
+          label="Movie"
+          ref={params.InputProps.ref}
+          startAdornment={params.InputProps.startAdornment}
+        />
+      )}
+    />
+  );
+};
+
+const EmptyTemplate: ComponentStory<typeof Autocomplete> = (args) => {
+  return (
+    <Autocomplete
+      {...args}
+      options={[]}
+      renderInput={(params) => (
+        <TextField
+          {...params}
+          endAdornment={params.InputProps.endAdornment}
+          hint="Select your favorite movie"
+          label="Movie"
+          ref={params.InputProps.ref}
+          startAdornment={params.InputProps.startAdornment}
+        />
+      )}
+    />
+  );
+};
+
+const MuiTemplate: ComponentStory<typeof Autocomplete> = (args) => {
+  return (
+    <Autocomplete
+      {...args}
+      options={top100Films}
+      renderInput={(params) => <TextFieldMui {...params} label="Movie" />}
+    />
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {};
+
+export const disabled = Template.bind({});
+disabled.args = {
+  disabled: true,
+  value: "The Godfather",
+};
+
+export const freeSolo = Template.bind({});
+freeSolo.args = {
+  freeSolo: true,
+};
+
+export const includeInputInList = Template.bind({});
+includeInputInList.args = {
+  includeInputInList: true,
+};
+
+export const loading = EmptyTemplate.bind({});
+loading.args = {
+  loading: true,
+};
+
+export const multiple = Template.bind({});
+multiple.args = {
+  multiple: true,
+};
+
+export const readOnly = Template.bind({});
+readOnly.args = {
+  readOnly: true,
+  value: "The Godfather",
+};
+
+export const MaterialSingle = MuiTemplate.bind({});
+MaterialSingle.args = {};
+
+export const MaterialMultiple = MuiTemplate.bind({});
+MaterialMultiple.args = {
+  multiple: true,
+};
```

Comment:
These are just for testing right?

Reply:
Correct, just to check what is rendered when using MUI's TextField against ours.